### PR TITLE
docs: Added _Creating a Podman machine with Podman Desktop_

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

docs: Added _Creating a Podman machine with Podman Desktop_

### What issues does this PR fix or reference?

refs: https://github.com/containers/podman-desktop/issues/2461

### CODE REVIEW

File: website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 11
+title: Custom Podman machine
+description: Podman Desktop can assist you to install custom Podman machine on Windows and macOS.
```
Comment (jeffmaury) [inline website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md]:
```suggestion
description: Podman Desktop can assist you to create custom Podman machine on Windows and macOS.
```

File: website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+tags: [podman-desktop, podman, installing, windows, macOS]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos, podman machine]
+---
+
+# Creating a Podman machine with Podman Desktop
+
+On macOS and Windows, running the Podman container engine requires running a Linux distribution on a virtual machine.
```
Comment (cdrage) [inline website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md]:
```suggestion
On macOS and Windows, running the Podman container engine requires running a Linux virtual machine.
```

File: website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+By default, Podman Desktop is initializing a Podman machine with a standard configuration.
```
Comment (cdrage) [inline website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md]:
```suggestion
By default, Podman Desktop initializes a Podman machine with a standard configuration.
```

I dont think we need this sentence maybe? Standard configuration gives no information as to what the standard config is?

File: website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Consider creating a custom Podman machine to:
```
Comment (cdrage) [inline website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md:15]: ❓ 

This section is a bit confusion to the reader. It's self explanatory, maybe a picture would say a thousand words instead?

File: website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+   1. **CPU(s)**:
+      Select the number of CPUs.
+   1. **Memory**:
+      Select the memory size.
+   1. **Disk size**:
+      Select the disk size.
+   1. **Image path (Optional)**:
+      Select a bootable image containing a virtual machine with Podman.
+   1. **Machine with root privileges**:
+      Enable to use the rootful connection by default.
```
Comment (cdrage) [inline website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md:42]: weve gotten question with regards to rootful / root privileges and why we need to enable them, I think we should link this section to an article / explain in more detail.

**Review by cdrage — DISMISSED** [review]

File: website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md (hunk 4 above)
Comment (themr0c) [reply website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md:15]: I don't understand. The screenshot below is the how. This section is the why, a screenshot would not make sense here.

File: website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md (hunk 5 above)
Comment (themr0c) [reply website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md:42]: This is explained in the prerequisite section.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**